test/snippets/expected/Automanage Create Custom Profile.snippetresult.json

```diff
@@ -27,7 +27,7 @@
 					"Backup/RetentionPolicy/DailySchedule/RetentionDuration/DurationType": "Days",
 					"WindowsAdminCenter/Enable": false,
 					"VMInsights/Enable": true,
-					"AzureSecurityCenter/Enable": true,
+                    "DefenderForCloud/Enable": true,
```

Comment:
use tabs

Reply:
Im no sure exactly how to fix this, everything looks good on my end: 
![image](https://user-images.githubusercontent.com/52691521/202611344-4a57bb3c-3635-4175-90b5-de9f6ecf86c4.png)